extensions/amp-story-360/0.1/amp-story-360.js

```diff
@@ -411,6 +420,20 @@ export class AmpStory360 extends AMP.BaseElement {
     }
   }
 
+  /** @private */
+  maybeActivateGlContext_() {
+    const distance = this.getPage_().getAttribute('distance');
```

Comment:
This won't be reliable, the `CURRENT_PAGE_ID` fires before the `distance` attribute is set because of how we chunk page navigation code.
I really appreciate the work you've done to remove 360 code from amp-story-page, but it might be necessary here to lose/restore context from `amp-story-page.setDistance()`

Another solution would be to use a `MutationObserver` and listen for `distance` attr updates.